omnibus/dependabot-omnibus.gemspec

```diff
@@ -5,24 +5,24 @@ Gem::Specification.new do |spec|
     Bundler.load_gemspec_uncached("../common/dependabot-common.gemspec")
 
   spec.name         = "dependabot-omnibus"
-  spec.summary      = "Meta-package that depends on all dependabot package " \
-                      "managers"
-  spec.version      = common_gemspec.version
-  spec.description  = common_gemspec.description
+  spec.summary      = "Meta-package that provides all the gems included in Dependabot Core"
+  spec.description  = "Dependabot-Omnibus provides all the gems included in Dependabot Core. " \
+                      "Dependabot provides automated dependency updates for multiple package managers."
 
   spec.author       = common_gemspec.author
   spec.email        = common_gemspec.email
   spec.homepage     = common_gemspec.homepage
   spec.license      = common_gemspec.license
 
-  spec.required_ruby_version = common_gemspec.required_ruby_version
-  spec.required_rubygems_version = common_gemspec.required_ruby_version
-
   spec.metadata = {
     "issue_tracker_uri" => common_gemspec.metadata["issue_tracker_uri"],
     "changelog_uri" => common_gemspec.metadata["changelog_uri"]
   }
 
+  spec.version = common_gemspec.version
+  spec.required_ruby_version = common_gemspec.required_ruby_version
+  spec.required_rubygems_version = common_gemspec.required_ruby_version
```

Comment:
I moved these under the `spec.metadata` for consistency of ordering with the other files, and then on the other files that resulted in moving `spec.require_path`/`spec.files` for consistency... but no actual changes in any of the values.

Reply:
only Solution for this is to make sure that the url you have passed is legal, with a proper protocol. The best way to do it is validating process